src/main/resources/application.properties

```diff
@@ -3,3 +3,4 @@ spring.data.mongodb.host=localhost
 spring.data.mongodb.port=27017
 security.jwt.token.secret_key=SECRET
 security.jwt.token.expire_length=3700000
+spring.profiles.active=docker
```

Comment:
let's revet this change. 
Let's keep default properties for default profile)

I recommend passing 'docker' profile inside Dockerfile via entry point
https://stackoverflow.com/a/55759349/2689114
Or via ENV. Actually, passing via env is more flexible. 
 
https://stackoverflow.com/a/55759349/2689114

Reply:
Done!